src/plugin/PluginBase.php

```diff
@@ -116,6 +132,13 @@ final public function onEnableStateChange(bool $enabled) : void{
 			$this->isEnabled = $enabled;
 			if($this->isEnabled){
 				$this->onEnable();
+				try{
+					foreach($this->getTranslations() as $translation){
+						$this->server->getLanguage()->merge($translation);
+					}
+				}catch(\InvalidArgumentException $e){
```

Comment:
Logical exceptions like this generally shouldn't be caught, as we don't know where they are coming from. If you want to catch an exception here, a specific exception should be introduced.